man/man1/fzf.1

```diff
@@ -1122,19 +1122,19 @@ The following variables are additionally exported to the preview commands.
 
 .SH EXTENDED SEARCH MODE
 
-Unless specified otherwise, fzf will start in "extended-search mode". In this
+Unless specified otherwise, fzf will start in "extended\-search mode". In this
 mode, you can specify multiple patterns delimited by spaces, such as: \fB'wild
 ^music .mp3$ sbtrkt !rmx\fR
 
 You can prepend a backslash to a space (\fB\\ \fR) to match a literal space
 character.
 
-.SS Exact-match (quoted)
+.SS Exact\-match (quoted)
```

Comment:
I'm now wondering whether we should escape these hyphens. Are these just the names in the explanations and not the keywords that are programmatically processed? Then, maybe we should allow the man page system to make those hyphens the real Unicode hyphens.

Reply:
I agree. Let's keep those hyphens as they are.

Reply:
Actually, some were escaped as `exact\-match` and some were left as `exact-match` in the PR branch, so I wanted to normalize it to either one. If you prefer `exact\-match`, you can use

[0001-man-Escape-exact-match-and-anchored-match.patch.txt](https://github.com/user-attachments/files/15948505/0001-man-Escape-exact-match-and-anchored-match.patch.txt)

Or, if you prefer `exact-match`, you can use

[0001-man-Revert-exact-match-and-anchored-match.patch.txt](https://github.com/user-attachments/files/15948513/0001-man-Revert-exact-match-and-anchored-match.patch.txt)

You can directly edit them by yourself, or if needed I can submit a followup PR.